stan/math/prim/scal/prob/bernoulli_logit_rng.hpp

```diff
@@ -0,0 +1,47 @@
+#ifndef STAN_MATH_PRIM_SCAL_PROB_BERNOULLI_LOGIT_RNG_HPP
+#define STAN_MATH_PRIM_SCAL_PROB_BERNOULLI_LOGIT_RNG_HPP
+
+#include <boost/random/bernoulli_distribution.hpp>
+#include <boost/random/variate_generator.hpp>
+#include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
+#include <stan/math/prim/scal/err/check_bounded.hpp>
+#include <stan/math/prim/scal/err/check_finite.hpp>
+#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/fun/constants.hpp>
+#include <stan/math/prim/scal/fun/inv_logit.hpp>
+#include <stan/math/prim/scal/fun/log1m.hpp>
+#include <stan/math/prim/scal/fun/value_of.hpp>
+#include <stan/math/prim/scal/meta/include_summand.hpp>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * A Bernoulli random number generator which takes as its argument the
+     * often more convenient logit-parametrization.
+     * 
+     * @param t logit-transformed probability parameter.
+     * @param rng pseudorandom number generator.
+     * @tparam RNG Random number generator type.
+     * @return Bernoulli(logit^{-1}(t)) generated random number, either 0 or 1.
+     */
+    template <class RNG>
+    inline int
+    bernoulli_logit_rng(const double t,
```

Comment:
1. remove `const`
2. don't know what happened with spacing, but move `RNG& rng` to the end of line 30. (Or make line 31 line up properly with line 30.